src/Components/DetailsPage.js

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { useParams, Link } from 'react-router-dom';
+import { fetchDetailsPage } from '../Redux/places/detailsPageSlice';
+import '../Placelist.css';
+import '../DetailsPage.css';
+
+function DetailsPage() {
```

Comment:
- Please, consider using arrow funcs according to Javascript ES6 good practices.